StudyPathTraker/StudyPathTraker/Custom Views/Generic/SPAlertCenter.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SPAlertCenter.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 6/11/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+class SPAlertCenter {
+    
+    static func showMessageWithTitle(_ title: String = "", message: String) -> UIAlertController {
```

Comment:
add this to a extension of  UIViewController, and rename to something for swifty.